pcdsdevices/gauge.py

```diff
@@ -0,0 +1,64 @@
+"""
+Standard classes for LCLS Gauges
+"""
+import logging
+from ophyd import EpicsSignal, EpicsSignalRO, Device, Component as Cpt
+
+logger = logging.getLogger(__name__)
+
+class BaseGauge(Device):
+    """
+    Vacuum gauge
+
+    A base class for a device with two limits switches controlled via an
+    external command PV. This fully encompasses the controls `Stopper`
+    installations as well as un-interlocked `GateValves`
+
+    Parameters
+    ----------
+    prefix : ``str``
+        Full Gauge base PV
+
+    name : ``str``
+        Alias for the gauge
+    """
+    # 
+    pressure = Cpt(EpicsSignalRO, ':PMON', kind='normal')
+    status = Cpt(EpicsSignalRO, ':STATUSMON', kind='normal')
+    pressure_status = Cpt(EpicsSignalRO, ':PSTATMON', kind='normal')
+    pressure_status_enable = Cpt(EpicsSignal, ':PSTATMSP', kind='normal')
+
+    def __repr__(self):
+        print('Gauge is %s'%self.status)
+        if self.status>0:
+            print('Pressure: %g'%self.pressure)
+
+class GaugePirani(BaseGauge):
+    """
+    Class for Pirani gauge
+    """
+    pass
+
+class GaugeColdCathode(BaseGauge):
+    """
+    Class for Cold Cathode Gauge
+    """
+    enable = Cpt(EpicsSignalRO, ':ENB_DO_SW', kind='normal')
+    #this should be combined.
+    tripSetpointRB = Cpt(EpicsSignal, ':PSTATSPDES', kind='normal')
+    tripSetpoint = Cpt(EpicsSignalRO, ':PSTATSPRBCK', kind='normal')
+
+class GaugeSet(Device):
+    """
+    Class for a gauge set with a Pirani and Cold Cathode Gauge
```

Comment:
Do you think replacing the Cold Cathode component with the Pirani is the cleanest? Why not have the prefix be the shared prefix and add them both? You should still be able to use the info contained in the comment below.